### DISCUSSION TITLE
having a model load at a specific angle

### DISCUSSION THREAD

**Discussion by yosun** [google/model-viewer#4810] [Q&A]
how do you have the model load at a different starting rotation?

**Comment by danilokrulj** [comment]
https://modelviewer.dev/examples/stagingandcameras/#cameraOrbit

Did you try looking this up or did you just decide to ask? Because this is very easy to find ;)

**Comment by yosun** [comment]
gemini couldn't answer that question! 
trying to add stuff for gemini to index eventually

**Reply by elalish** [reply]
Or slightly different: https://modelviewer.dev/docs/#entrydocs-scenegraph-attributes-orientation

PS - our docs page is fully searchable with the browser's ctrl-F.
